tensorflow/lite/micro/kernels/kernel_util.h

```diff
@@ -39,6 +40,13 @@ inline TfLiteEvalTensor* GetMutableEvalInput(const TfLiteContext* context,
 // Returns the TfLiteEvalTensor struct for a given input index in a node.
 inline const TfLiteEvalTensor* GetEvalInput(const TfLiteContext* context,
                                             const TfLiteNode* node, int index) {
+  const int tensor_index = tflite::ValidateTensorIndexing(
+      context, index, node->inputs->size, node->inputs->data);
+
+  if (tensor_index < 0) {
+    return nullptr;
+  }
```

Comment:
let's move this check to GetMutableEvalInput

Reply:
ok